index.html

```diff
@@ -3,25 +3,30 @@
     <head>
         <meta charset="utf-8">
         <title> Taher Haggui </title>
-        <link rel="stylesheet" type="text/css" href="styles/style.css">
         <link rel="stylesheet" type="text/css" href="styles/template2.css">
         <link rel="preconnect" href="https://fonts.googleapis.com">
         <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
         <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;1,100;1,600&display=swap" rel="stylesheet">
         <meta name="viewport" content="width=device-width, initial-scale=1">
     </head>
     <body>
+      <section class="home-page">
         <header>
           <nav id="nav">
             <a class="logo" href="#">My Logo</a>
 
             <nav><img src="statics/images/Union.png" alt="Menu"></nav>
+            <ul class="internal-link">
+               <li><a href="#my_work">Portfolio</a></li>
+               <li><a href="#about_me">About</a></li>
+               <li><a href="#form-section"> Contact </a></li>
+            </ul>
 
           </nav>
         </header>
         <section id="home_presentation">
```

Comment:
- [OPTIONAL] Kindly style the bottom of the `home_presentation` section as the original design. The `border-bottom has a rounded corner.
![Captura de pantalla_2021-09-24_21-18-57](https://user-images.githubusercontent.com/53111464/134754791-e831eeb1-fda7-4461-8428-05863b797f6c.png)